### PR TITLE
Proposal to add moderator actions

### PR DESCRIPTION
**Pull request by rbaker2019**
This is an initial attempt to add moderator functionality to the node-red-contrib-node-reddit collection of nodes. The following changes have taken place:

Housekeeping:
- Bump up dependencies to latest versions
- Reformat code (VSCode defaults)
- Minor code cleanup (comment and quotation mark consistency)

Big changes:
- Add GetModqueue node and documentation. This returns payload objects (formatted via the copyPropertiesExceptMethods method) representing entries in the modqueue for a given subreddit.
- Added icon for Moderator actions

When you have some time, would you mind reviewing this change to see if this is something we could develop further and incorporate in to your module?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Logs
```
Comment (jcostello93) [inline .gitignore:1]: This is adding quite a few files/directories that aren't 100% relevant to this project. Would you be able to add those to your global `.gitignore` instead?

File: node-reddit.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ module.exports = function (RED) {
 	});
 
 
+	/***** Get Node *****/
```
Comment (jcostello93) [inline node-reddit.js:85]: 👍

**Comment by jcostello93** [comment]
Hey, thanks for contributing! Happy to add some more functionality to these.

One thing I noticed is that the formatting for the getmodqueue node is a bit off: 

image

In general, I'd prefer to add this new functionality to the `get` node. I know there's a lot going on in that function but it seems more intuitive from a user perspective. 

When we first developed this, we decided to have fewer nodes and group them by CRUD-y actions. That's why there's currently the one `get` node with a few dropdowns based on user input.

**Comment by rbaker2019** [comment]
Thanks! After thinking about it, the moderator actions might be better suited as an addon package that extends the core functionality of your nodes. I'll try to think the design through a little more and see if I can get something going that can build on your workflow without overloading the core stream/get/create/etc nodes.

**Pull request closed without merging**